src/main/java/de/fraunhofer/aisec/cpg/graph/Annotation.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020, Fraunhofer AISEC. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+
+package de.fraunhofer.aisec.cpg.graph;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Annotation extends Node {
+
+  // should be extended later into annotation members
+  private List<Expression> values;
+
+  public List<Expression> getValues() {
+    return values;
+  }
+
+  public void setValues(List<Expression> values) {
+    this.values = values;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof Annotation)) {
+      return false;
+    }
+
+    Annotation that = (Annotation) o;
+    return super.equals(that) && Objects.equals(values, that.values);
+  }
+
+  @Override
+  public int hashCode() {
+    return super.hashCode();
```

Comment:
Why override the hashcode function here if you just call super? Should we include the values list in the hash? Otherwise I would just remove the overriding function

Reply:
I was basically copying the way all other nodes do it (which in my opinion does not make sense at all). I would suggest keeping it consistent with the others and see if we want to change it globally.

Reply:
Ah alright, that makes sense. I'll open another issue for that 👍